Server/routes/route.js

```diff
@@ -1,46 +1,42 @@
 import express from 'express';
-import parcelController from '../controllers/parcelcontroller';
-import usersControllers from '../controllers/usersController';
+import parcelController from '../controllers/ParcelController';
+import usersControllers from '../controllers/UsersController';
 import middlewares from '../middlewares/index';
 
-const { validateParcels, validateRegister, validateLogin, validateToken, validateSuperAdmin, validateAdmin } = middlewares;
+const { validateParcels, validateRegister, validateLogin, validateToken, validateSuperAdmin, validateAdmin, validateLocationUpdate } = middlewares;
+
+const { updateParcelDestination, cancelParcelOrder, registerUser, login, createAdmin  } = usersControllers;
+
+const { createNewParcel, getParcelsByUser, getAllParcels, getAllParcelsBySpecificUser, updateParcelStatus, updateParcelPresentLocation } = parcelController;
 
 const router = express.Router();
 
 // this is for the home route
 router.get('/', (req, res) => {
   res.status(200).send('welcome to SendIt');
 });
-
 // this is the route for creating parcels
-// CREATE PARCELS
-router.post('/parcels', validateParcels, validateToken, parcelController.createNewParcel);
+router.post('/parcels', validateParcels, validateToken, createNewParcel);
 // this is the route to get all parcels for a user
-// GET ALL PARCELS
-router.get('/parcels/user', parcelController.getParcelsByUser);
-// this is the route to get a specific parcel
-// GET A SPECIFIC PARCEL
-// router.get('/parcels/:id', parcelController.getSpecificParcel);
-// this is to change the dropofflocation of a parcel
-// PUT IN A NEW DROPOFFLOCATION
-router.put('/parcels/:pid/newlocation', validateToken, validateAdmin, parcelController.updateParcelPresentLocation);
+router.get('/parcels/user',validateToken, getParcelsByUser);
```

Comment:
This route is wrong consult your rubric for the right route.
Also, the controller to this route should not be in the the `ParcelController`  but `UsersController`

Reply:
it has been fixed